packages/vscode-plugin/src/tests/suite/integration.test.ts

```diff
@@ -96,4 +97,13 @@ describe('Integration >', () => {
 		await workspace.fs.writeFile(markdownUri, markdownContent);
 		await openFile('integration.md');
 	});
+
+	it('does nothing for untitled', async () => {
+		const untitledUri = await openUntitled('Errorz');
+
+		compareActualVsExpectedDiagnostics(
+			getActualDiagnostics(untitledUri),
+			createExpectedDiagnostics()
+		);
+	});
```

Comment:
Can you move this after the "gives correct diagnostics" test, just so they're better grouped together